### ISSUE TITLE
install issue

### ISSUE THREAD

**Issue by vim-zz** [nrc/cargo-src#234]
i am trying to install it using `cargo install cargo-src` but get the following error:

```
Compiling cargo_metadata v0.5.4
error[E0061]: this function takes 3 parameters but 2 parameters were supplied
  --> src/highlight.rs:46:46
   |
46 |         let mut classifier = Classifier::new(lexer::StringReader::new(&sess, fm), sess.codemap());
   |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 3 parameters

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: failed to compile `cargo-src v0.1.3`, intermediate artifacts can be found at `/var/folders/mk/tns5tyxd67jcbmb2k_d3k96w0000gn/T/cargo-installFY9YL1`
```

i am using `rustc 1.28.0-nightly (cb20f68d0 2018-05-21)`

**Comment by nrc** [comment]
This is fixed on master but broken in the latest release. I'll issue a new release today

**Comment by nrc** [comment]
published 0.1.4
